hack/local-cmo.sh

```diff
@@ -1,28 +1,111 @@
 #!/usr/bin/env bash
+set -eu -o pipefail
+
 # Script will:
 # 1. Patch CVO to exclude CMO from being managed
 # 2. Scale in-cluster CMO to 0
 # 3. Try to run CMO locally (CMO needs to be built before starting)
 
-set -eu
 
-# Go to the root of the repo
-cd "$(git rev-parse --show-cdup)"
+validate() {
+  local ret=0
+
+  [[ -z ${KUBECONFIG+xxx}  ]] && {
+    echo "ERROR: KUBECONFIG is not defined"
+    ret=1
+  }
+
+  [[ -x operator ]] || {
+    echo "ERROR: Failed to find 'operator' binary."
+    echo "       Did you run 'make run-local' or 'make build' ?"
+    ret=1
+  }
+
+  gojsontoyaml  --help 2>/dev/null || {
+    echo "ERROR: gojsontoyaml not found. See: https://github.com/brancz/gojsontoyaml#install"
+    ret=1
+  }
+
+  jq --version >/dev/null || {
+    echo "ERROR: jq not found. See: https://stedolan.github.io/jq/download/"
+    ret=1
+  }
+
+  return $ret
+}
+
+kc() {
+  kubectl -n openshift-monitoring "$@"
+}
```

Comment:
What is the benefit of this wrapper? Considering it is used twice and the script still needs to use `kubectl` directly in other cases.

Reply:
it avoids having to add `-n openshift-monitorng` (so typos are avoided ;-p) ... but more importantly `kc` is context switch to  indicate that the commands are specific to `openshift-monitoring` namespace as opposed to something that is across the cluster (CVO)